packages/component/src/Localization/Localize.js

```diff
@@ -152,8 +152,6 @@ function localize(text, language, ...args) {
   return string || text;
 }
 
-export default connectToWebChat(({ language }) => ({ language }))(({ args, language, text }) =>
-  localize(text, language, ...(args || []))
-);
+export default ({ args, text }) => useLocalize(text, ...(args || []));
```

Comment:
This component seems unnecessary...

Reply:
Yap, but to maintain backward compatibility.